src/updaters/dotnet/readme.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {checkpoint, CheckpointType} from '../../util/checkpoint';
+import {GitHubFileContents} from '../../github';
+import {Update, UpdateOptions, VersionsMap} from '../update';
+
+export class ReadMe implements Update {
+  path: string;
+  changelogEntry: string;
+  version: string;
+  versions?: VersionsMap;
+  packageName: string;
+  create: boolean;
+  contents?: GitHubFileContents;
+
+  constructor(options: UpdateOptions) {
+    this.create = false;
+    this.path = options.path;
+    this.changelogEntry = options.changelogEntry;
+    this.version = options.version;
+    this.packageName = options.packageName;
+    this.contents = options.contents;
+  }
+
+  updateContent(content: string): string {
+    checkpoint(`updating ${this.path}`, CheckpointType.Success);
+    return content.replace(
```

Comment:
It looks like this would cause problems if the README contained any version number other than the one we want to replace. I think we'd probably want to be more specific than this.